video/agon_audio.h

```diff
@@ -106,7 +111,7 @@ void initAudio() {
 // Channel enabled?
 //
 bool channelEnabled(uint8_t channel) {
-	return channel < MAX_AUDIO_CHANNELS && audioChannels[channel];

```

Comment:
this version of checking for the existence of a channel worked, but if this was called for a non-existent channel it would create an entry in the audioChannels map with a null pointer for the non-existent channel

that entry in the map would then cause issues if audioChannels was iterated over 🤦